### PR TITLE
Release v2 of gem

### PR DESCRIPTION
**Pull request by luke-hill**
Updated some comment docs and release v2

### CODE REVIEW

**Review by mrdaniellewis — APPROVED** [review]
Looks good

**Comment by luke-hill** [comment]
So I've learnt two new things @mrdaniellewis 

order random is a must in every project.
`a_hash_including` is a nice matcher to have!

File: lib/ca_testing/drivers/v4/local.rb [hunk 1]
```diff
@@ -43,7 +48,7 @@ def capabilities
         # This is required to make local drivers work exclusively with Safari TP
         # This is required in V13 of Safari as the driver there is notoriously flaky
         # In V12 it doesn't hinder it
-        # We don't support Safari V11!
+        # Safari V11 is unsupported.
```
Comment (mrdaniellewis) [inline lib/ca_testing/drivers/v4/local.rb:51]: I wanted to use Safari 11 in the Casebook smoke tests, as we still have some users.  But yes, it was completely broken.  Just ended up using 13.

**Comment by mrdaniellewis** [comment]
We do the random order in the unit tests, but not the feature tests 😬, probably should change that

File: lib/ca_testing/drivers/v4/local.rb (hunk 1 above)
Comment (luke-hill) [reply lib/ca_testing/drivers/v4/local.rb:51]: Funnily enough safari have basically said v12 is also "unsupported" so technically both are unsupported. But then you get into a minefield.

The safaridriver v12 I consider to be more performant and accurate than the v13 one. Primary reason being the v12 driver was one that contained both JWP and W3C stuff, and I theorise (It's closed source), that there are fallbacks e.t.c. in there that make things "work better"

safaridriver v13 just fails and falls over so frequently it was a nuisance when I first joined, so I just reverted to using safaridriver v12 again.

I'm midway through trying to get safaridriver v14 to work reliably, but outside of using it sporadically, it's falling over a lot (Although I believe that may be more browserstacks end).

**Review by david-lake — APPROVED** [review]

**Pull request merged**